### PR TITLE
Copy files from Admissions Trial

### PR DESCRIPTION
**Pull request by dd-obua**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+
+    <script src="js/index.js"></script>
```
Comment (dd-obua) [inline index.html:40]: Kindly remove the unused scripts from your code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        <figcaption id="img-caption">Tribute Image</figcaption>
+        <img id="image" src="img/image.jpg" />
+      </div>
+      <p id="tribute-info">Subject of the Tribute</p>
+      <a
+        href="https://www.thegravesendflorist.co.uk/design-20179416/dummy-tribute.htm"
+        id="tribute-link"
+        target="_blank"
+        >Link to Tribute</a
+      >
```
Comment (dd-obua) [inline index.html:23]: For semantic reasons, kindly wrap these lines of code in a <main> element.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
+* {
+  font-size: 24px;
+  font-family: Arial, Helvetica, sans-serif;
```
Comment (dd-obua) [inline css/style.css:3]: Please, consider setting these properties inside body element

**Review by dd-obua — COMMENTED** [review]
The entire code looks good. Just a few of those little changes will make you a better professional.

File: index.html (hunk 1 above)
Comment (uche-inyama) [reply index.html:40]: When you request for a change on a line(s) of code, kindly give the reason(s), for requesting for such changes.

### :slightly_smiling_face:

Comment (uche-inyama) [reply index.html:40]: When you request for a change on a line(s) of code, kindly give the reason(s), for requesting for such changes.

### :slightly_smiling_face:

File: css/style.css (hunk 3 above)
Comment (uche-inyama) [reply css/style.css:3]: When you request for a change on a line(s) of code, kindly give the reason(s), for requesting for such changes.

### :slightly_smiling_face:

**Review by uche-inyama — APPROVED** [review]
Hi @dd-obua :wave:  

Good job, let's move on with the next project!

## Highlights

- [X] You have shown you understand how to work with github code review features.
- [x] You have identified inconsistencies in the css and html files.
- [x] Your communication was always polite and you used a suggestive tone.
- [x] In some of your comments, you did not mention the reason(s) why you have requested for a change. Kindly do so whenever you are making a code review.

### Kindly go through the comments I have left :point_up_2: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (uche-inyama) [reply index.html:40]: When you request for a change on a line(s) of code, kindly give the reason(s), for requesting for such changes.

### :slightly_smiling_face:

Comment (uche-inyama) [reply index.html:40]: When you request for a change on a line(s) of code, kindly give the reason(s), for requesting for such changes.

### :slightly_smiling_face:

File: css/style.css (hunk 3 above)
Comment (uche-inyama) [reply css/style.css:3]: When you request for a change on a line(s) of code, kindly give the reason(s), for requesting for such changes.

### :slightly_smiling_face:

**Pull request merged**